### PR TITLE
Create Seeder Category

### PR DESCRIPTION
**Pull request by Web-Ange**

### CODE REVIEW

File: database/factories/OrderFactory.php [hunk 1]
```diff
@@ -15,7 +15,7 @@ public function definition()
     {
         return [
             'user_id' => $this->faker->unique()->user id(),
-            'order_date' => '$2y$10$92IXUNpkjO0rOQ5byMi.Ye4oKoEa3Ro9llC/.og/at2.uheWG/igi', // password
+            'order_date' => $faker->dateTime();
```
Comment (benotoho) [inline database/factories/OrderFactory.php:18]: This should be `$this->faker->dateTime()`. Did you try running the seeder to be sure there are no errors?

**Comment by Web-Ange** [comment]
i did

**Comment by Web-Ange** [comment]
i have change that code so many times i don't know why it keeps go back  to
that

**Comment by Web-Ange** [comment]
i have change all that

**Pull request merged**